internal/controller/storageclaim_controller.go

```diff
@@ -550,6 +636,20 @@ func (r *StorageClaimReconciler) getCephRBDVolumeSnapshotClass() *snapapi.Volume
 	return volumesnapshotclass
 }
 
+func (r *StorageClaimReconciler) getCephDriverVolumeGroupSnapshotClass(
+	labels map[string]string, annotations map[string]string, driver string) *groupsnapapi.VolumeGroupSnapshotClass {
```

Comment:
We could add the labels/annotations directly in the CreateOrReplace

Reply:
We might not need this function, we could have everything under CreateOrReplace.